package.json

```diff
@@ -14,6 +14,7 @@
     "bump": "^0.2.5",
     "del": "^2.2.0",
     "gulp": "^3.9.1",
+    "gulp-angular-templatecache": "^2.0.0",
```

Comment:
This change is correct, i forgot to add it when i first built the component
https://github.com/bypasslane/angular-material-expander/blob/95efed78c059bb0bd66a5bdbc8c035ff3cbebcb8/gulpfile.js#L13